placenames.mss

```diff
@@ -43,22 +31,27 @@
 }
 
 .state {
-  [zoom >= 5][way_pixels > 3000][way_pixels < 196000] {
+  [zoom >= 5][way_pixels > 3000][way_pixels < 192000] {
```

Comment:
I would approve of this minor change, but `#admin-text` needs to be changed to match [admin_level = '4'][way_pixels >= 196000] currently, would need to change to [way_pixels >= 192000]